README.md

```diff
@@ -1,24 +1,174 @@
+</details> -->
+
+<details>
+<summary>Frontend</summary>
+  <ul>
+    <li><a href="https://www.react.dev/">React</a></li>
+  </ul>
+
+<summary>Backend</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.ruby.org/">Ruby</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://www.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Connect rails with react** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+[Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+
+### Setup
+
+Clone or download the Collectify repository to your local machine.
+```sh
+  cd my-folder
+  git clone https://github.com/fpsapc/hello-rails-react.git
+```
+Open your terminal and navigate to the cloned project directory.
+
+### Install
+
+This project does not require additional dependencies, just Ruby interpreter. However, it uses RuboCop as a linter and you should set it up as well.
+
+```sh
+bundle install
+npm install
+```
+
+
+### Usage
+ 
+To run the project, navigate to the project directory and execute the following command:
+
+```sh
+  bundle install
+  rails db:create db:migrate 
+  npm run build
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Run the following command
+```sh
+No tests available for this project/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Salman Ahmad Khan**
+- GitHub: [@githubhandle](https://github.com/fpsapc)
+- Twitter: [@twitterhandle](https://twitter.com/fpsapc)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/salman-ahmad1987 )
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **This project is for just testing purpose**
+- There are no future features in it right now.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+please feel free to submit a pull request or open an issue on the project repository.
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give us a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Special thanks to microverse team for their guidlines.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
- [ ] You have done a good job🥇🥇, however, let's make our README file more professional. Currently, the link is broken, it giving a `404` message 😿😢. Kindly, fix it.

![Screenshot 2023-08-16 142227](https://github.com/fpsapc/hello-rails-react/assets/57408419/e6d2809e-f5b4-47b0-8eb4-7620773e741e)